### DISCUSSION TITLE
Monthly meetings

### DISCUSSION THREAD

**Discussion by KaiVolland** [geostyler/geostyler#1341] [Show and tell]
Starting in March 2021, there will be monthly meetings where GeoStyler developers and users can get together to talk about new features, roadmap status, open issues, and all other GeoStyler-related things.

The meetings will always be held on the first tuesday of the month (13:00 GMT).

- March (2.3.2021)
- April (6.4.2021)
- May (4.5.2021)
- …

Everybody is very welcome to participate. Please leave a comment so we can send you an invitation link to the call.

**Comment by hwbllmnn** [comment]
Count me in!

**Comment by dnlkoch** [comment]
I am also in.

**Comment by weskamm** [comment]
me2

**Comment by chrismayer** [comment]
In

**Comment by jansule** [comment]
+1

**Comment by marcjansen** [comment]
Please invite me as well

**Comment by ahennr** [comment]
👋🏼

**Comment by annarieger** [comment]
me too please :)
